Specification/BrAPI-Schema/BrAPI-Germplasm/SeedLot.json

```diff
@@ -167,7 +160,7 @@
         },
         "ContentMixture": {
             "properties": {
-                "mixtureDbId": {
+                "contentMixtureDbId": {
                     "description": "The unique DbId for a cross contained in this seed lot",
                     "type": [
                         "null",
```

Comment:
To be consistent with other parts of the JSON Schema crossDbId and crossName should be replaced by a single property cross, with a link to cross. Similarly for germplasmDbId and germplasmName, with germplasm property and link to Germplasm

Reply:
So basically an association to the referenced model, right?
>single property cross
So one-to-one relationship?